tests/test_diffeq/test_odesolver.py

```diff
@@ -1,87 +0,0 @@
-import unittest
-
```

Comment:
the whole test suite is quite redundant. It does not test anything that is not covered by the meaningful tests, for example those for the ODE filter.

Reply:
(this test suite has been a thorn in my side for a while now, because for every refactoring in the ODESolver interface, I needed to meaninglessly change the tests here.)